backend/database/test_connection.go

```diff
@@ -5,11 +5,10 @@ package database
 
 import (
 	"database/sql"
-	"io/ioutil"
-	"testing"
-
 	"github.com/go-sql-driver/mysql"
 	"github.com/stretchr/testify/require"
+	"os"
```

Comment:
It's Go convention to separate out std imports from external imports by skipping a line in between. Within a block goimports/gofmt/etc will alphabetize.

Reply:
Thanks for spotting that, I've fixed the ordering on the files I updated.